src/Widget.php

```diff
@@ -78,7 +77,8 @@ final public static function end(): string
     /**
      * Creates a widget instance.
      *
-     * @param array|callable|string $config The parameters for creating a widget.
+     * @param array $constructorArguments The constructor arguments.
+     * @param array $config The parameters for creating a widget.
```

Comment:
I think we need to referene definitions via `@link` here.

Reply:
done